.mergify.yml

```diff
@@ -14,6 +14,8 @@ pull_request_rules:
       - "#approved-reviews-by>=2"
       - "#changes-requested-reviews-by=0"
       - "status-success=continuous-integration/travis-ci/pr"
+      - "status-success=DCO"
+      - "status-success=commitlint"
```

Comment:
What does `commitlint` check? Can you link to a description in the mergify docs?

Reply:
commitlint is a bot added in cephcsi repo and the rules are valid, you can check the rules are valid or not here https://github.com/ceph/ceph-csi/pull/1017/checks?check_run_id=648486513

Reply:
Well, yes, I understand that part. But what kind of requirements does `commitlint` have on the commit messages?

Reply:
https://github.com/ceph/ceph-csi/pull/1015 adds the required details on how the commit message should be like

Reply:
Right, this should probably all have been done in the same PR :man_shrugging: